question21/main.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+const wrapper = document.getElementById('js-wrapper');
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const showLoadImg = () => {
+  const loadImg = createElementWithClassName("img", "loading");
+  loadImg.src = "./img/loading-circle.gif";
+  loadImg.id = 'js-loading';
+  wrapper.appendChild(loadImg);
+} 
+
+const removeLoadImg = () => {
+  document.getElementById('js-loading').remove();
+} 
```

Comment:
nits
Arrow functions can omit {} for a single line.
```suggestion
const removeLoadImg = () => document.getElementById('js-loading').remove();
```

Reply:
Thank you for your review!
Oh yes! Fixed so please confirm.
fixed commit : https://github.com/csakyo/homework/pull/36/commits/a1434353d447d79f361194ca29e9917bf3cb29fc